frontend/src/features/admin/users/ManageUsers.tsx

```diff
@@ -26,18 +26,47 @@ import styled from 'styled-components';
 import useCodeLookups from 'hooks/useLookupCodes';
 import { isEmpty } from 'lodash';
 import _ from 'lodash';
+import variables from '_variables.module.scss';
+import { FaFileExcel } from 'react-icons/fa';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+import download from 'utils/download';
+import queryString from 'query-string';
+import { ENVIRONMENT } from 'constants/environment';
+import { IPaginateParams } from 'constants/API';
 
 const TableContainer = styled(Container)`
   margin-top: 10px;
   margin-bottom: 40px;
 `;
 
-export const ManageUsersPage = () => {
+const FileIcon = styled(Button)`
```

Comment:
Continuous improvement, would be nice to start building a library of these rather then recreating them as one-offs. That way we can more consistently make cross-application style changes.

Reply:
no change required here, this is a pattern we are doing everywhere.

Reply:
Yes it would be nice to have a library of buttons!  Tech-debt